src/Fixer/Phpdoc/PhpdocTrimFixer.php

```diff
@@ -114,15 +115,15 @@ private function fixStart($content)
      */
     private function fixEnd($content)
     {
-        return preg_replace(
+        return PregWrapper::replace(
```

Comment:
I do understand that we gonna replace all usage of `preg_*` ?

Reply:
Yes: a new AutoReview will ensure `preg_*` calls outside PregWrapper are forbidden and must use the proxy instead.